bower.json

```diff
@@ -21,6 +21,6 @@
         "spec"
     ],
     "dependencies": {
-        "moment": "~2.10.3"
+        "moment": "~2.12"
```

Comment:
@benjaminoakes Note that you can't *just* drop `build/` for bower, since it's the only version of the JS that gets into bower right now. But if you remove `lib/` from the `ignore` section in `bower.json`, and if you're intending to keep this ES5 only, then sure.. :)

I'll update the PR with the requested changes today..